tutorials/selenium/README.md

```diff
@@ -0,0 +1,14 @@
+# JxBrowser and Selenium WebDriver
+
+These projects demonstrate how to integrate a JxBrowser-based application with Selenium WebDriver.
```

Comment:
This tutorial demonstrates how to create a simple Selenium Chrome WebDriver app that is configured to access a web page loaded in a desktop Java app using JxBrowser on Windows.